app/src/main/java/de/xikolo/views/ExoPlayerVideoView.kt

```diff
@@ -176,9 +202,16 @@ open class ExoPlayerVideoView : PlayerView {
         this.uri = uri
         videoMediaSource =
             if (isHls) {
-                HlsMediaSource.Factory(dataSourceFactory).createMediaSource(uri)
+                HlsMediaSource.Factory(dataSourceFactory)
+                    .setAllowChunklessPreparation(true)
```

Comment:
That's an `HLS` feature?

Reply:
Yes, it is an optimization allowing HLS videos to load faster by not downloading a chunk first to check codecs etc., because this information is already in the master playlist. Does not matter in the end, because HLS is yet only enabled in debug mode.